app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,25 @@
+package ar.com.wolox.android.example.ui.home
+
+import ar.com.wolox.android.example.ui.home.News.Create.CreateNewsFragment
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+import javax.inject.Inject
+
+/**
+ * Home activity
+ */
+@Suppress("CAST_NEVER_SUCCEEDS")
```

Comment:
A bit worried for this warning . we should remove it and then check what it's causing it.